dxr/query.py

```diff
@@ -326,6 +327,62 @@ def generic_visit(self, node, visited_children):
         return visited_children or node
 
 
+# TODO next next: write some tests
+class FilterAggregator(Filter):
+    """Filter class that acts by constructing the union of some subset of the
+    filters of all currently enabled plugins.
+
+    """
+
+    def __init__(self, term, plugins, condition=None):
```

Comment:
I like that you made `condition` part of the constructor. There's no reasonable default, and this makes sure you can't leave it out.